source/gui/settingsDialogs.py

```diff
@@ -1112,23 +1112,22 @@ def makeSettings(self, settingsSizer):
 		self.lineNumberCheckBox.SetValue(config.conf["documentFormatting"]["reportLineNumber"])
 		settingsSizer.Add(self.lineNumberCheckBox,border=10,flag=wx.BOTTOM)
 		sizer=wx.BoxSizer(wx.HORIZONTAL)
-		# Translators: This is the label for a combobox in the
+		# Translators: This is the label for a combobox controlling the line indentation in the
 		# Document  Formatting  dialog (possible choices are Off, Speech, Tones, or Both.
-		sizer.Add(wx.StaticText(self,wx.ID_ANY,label=_("Report Line &Indents With:")))
+		sizer.Add(wx.StaticText(self,wx.ID_ANY,label=_("Report Line &Indentation With:")))

```

Comment:
nit: Lower case for "line indentation". I think you missed this in the last review. :)